bin/apisix

```diff
@@ -284,6 +284,12 @@ http {
         }
         {% end %}
 
+        {% if server then %}
+        {% for directive, directive_val in pairs(server) do %}
+        {*directive*} {*directive_val*};
+        {% end %}
+        {% end %}
+
```

Comment:
I don't think put server configures between `location` is a good idea, you should put them to the begin of `server`.

Reply:
moved to the begin of `server`